docs/env.md

```diff
@@ -114,6 +114,17 @@ Below is a list of currently supported environment variables:
   - Optional parameter of plan intervention types whose tasks should be generated on opensrp
   - Options: FI, IRS, IRS_Lite, MDA, MDA-Point, Dynamic-FI, Dynamic-IRS and Dynamic-MDA
 
+- **REACT_APP_HIDE_PLAN_FORM_FIELDS_ON_EDIT**
+
+  - Optional parameter of plan intervention types whose some fields will be hidden when the plan status is draft, active or completed.
+  - Options: caseNum, interventionType, jurisdictions, fiReason, fiStatus, title, triggersAndConditions activityActionDefinitionUri, activityActionDescription, activityActionTitle, activityActionReason activityTimingPeriodStart, activityTimingPeriodEnd, activityGoalPriority, activityGoalValue
+
+- **React_APP_CASE_TRIGGERED_DRAFT_EDIT_ADD_ACTIVITIES**
```

Comment:
Could you explain why we need to treat case triggered plans specially?

Reply:
Base on the request [here](https://github.com/onaio/reveal-frontend/issues/1378#issuecomment-773263684) adding and editing of the activities is only supposed to happen for case triggered plans only.